idd/Energy+.idd.in

```diff
@@ -61352,19 +61521,43 @@ NodeList,
   A492,A493,A494,A495,A496,A497,A498,A499,A500,A501; \note fields as indicated
 
 OutdoorAir:Node,
-       \memo This object sets the temperature and humidity conditions
-       \memo for an outdoor air node.  It allows the height above ground to be
-       \memo specified.  This object may be used more than once.
-       \memo The same node name may not appear in both an OutdoorAir:Node object and
-       \memo an OutdoorAir:NodeList object.
+      \memo This object sets the temperature and humidity conditions
+	  \memo for an outdoor air node.  It allows the height above ground to be
+	  \memo specified.  This object may be used more than once.
+	  \memo The same node name may not appear in both an OutdoorAir:Node object and
+	  \memo an OutdoorAir:NodeList object.
+	  \memo This object defines local outdoor air environmental conditions.
   A1,  \field Name
        \required-field
        \type node
-  N1 ; \field Height Above Ground
+  N1,  \field Height Above Ground
        \note A value less than zero indicates that the height will be ignored and the weather file conditions will be used.
        \type real
        \units m
        \default -1.0
+  A2 , \field Drybulb Temperature Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Schedule values are real numbers, -100.0 to 100.0, units C 
+  A3 , \field Wetbulb Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Schedule values are real numbers, -100.0 to 100.0, units C
+  A4 , \field Wind Speed Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Schedule values are real numbers, 0.0 to 40.0, units m/s
+  A5 , \field Wind Direction Schedule Name 
+       \type object-list
+       \object-list ScheduleNames
+       \note Schedule values are real numbers, 0.0 to 360.0, units degree
+  A6 ; \field Wind Pressure Coefficient Curve Name 
```

Comment:
If the wind Cp curve is added here, then the relative wind angle and symmetric options need to be added as well. Otherwise, some curves that are valid for external nodes will not be valid for these nodes. That's inconsistent and needs to be fixed.